extensions/2.0/Khronos/KHR_lights_environment/README.md

```diff
@@ -0,0 +1,261 @@
+
+This extension specifies exactly one way to format and reference the environment map to be used, the goals of this are two-fold.  
+First, it makes implementing support for this extension easier.  
+Secondly, it ensures that rendering of the image-based lighting is consistent across runtimes.
+
+A conforming implementation of this extension must be able to load the environment data and render the PBR materials using this lighting.  
+
+Cubemap environment light images are declared as an array of images in the extension, this extension is then placed in the glTF root.  
+These images shall be supplied using a KTX V2 file containing at least one cubemap.  
+
+### Pre Filtering Of Cubemaps
+
+The environment light is defined by a cubemap, this is to simplify realtime implementations as these are likely to use cubemap texture format.  
+Cubemaps shall be supplied without pre-filtered mip-maps for roughness values > 0, pre-filtering shall be done by the client.  
+[See Specular radiance cubemaps](#specular-radiance-cubemaps)  
+
+
+## Declaring An Environment Light
+
+The KHR_lights_environment extension defines an array of environment light cubemaps at the root of the glTF, each scene can reference one these lights.    
+Each environment light definition consists a single cubemap that describes the specular radiance of the scene, the l=2 spherical harmonics coefficients for irradiance, luminance factor and bounding box for localized cubemap.  
+
+The cubemap is defined as an integer reference to one of the texture sources declared in the this extension. 
+This texture source shall contain a cubemap at the layer defined by the texture.   
+
+When this extension is used, images shall use `image/ktx2` as mimeType.  
+The texture type of the KTX v2 file shall be 'Cubemap'  
+
+The following will declare one cubemap to be referenced by a scene.  
+
+
+```json  
+
+"asset": {
+"version": "2.0"
+},
+"extensionsUsed": [
+    "KHR_lights_environment"
+],
+
+"extensions": {
+    "KHR_lights_environment" : {
+        "cubemaps": [
+            {
+                "source": 0
+                "layer": 0
+            }
+        ],
+        "images": [
+            {
+                "name": "environment cubemap 0",
+                "uri": "cubemap0.ktx2",
+                "mimeType": "image/ktx2"
+            }
+        ],  
+        "lights": [
+            {
+                "name": "environment light 0",
+                "irradianceCoefficients": [...3 x 9 array of floats...],
+                "boundingBoxMin": [-100, -100, -100],
+                "boundingBoxMax": [100, 100, 100],
+                "cubemap": 0,
+            }
+        ]
+    }
+}
+```
+
+## Specular Radiance Cubemaps
+
+The cubemap used for specular radiance is defined as a cubemap containing separate images for each cube face.  
+The data in the maps represents illuminance in candela per square meter.  
```

Comment:
I'm not sure how to interpret `cd / m²` —  wouldn't we need to know the physical area represented by a pixel to use that? Perhaps `cd / sr` (sr = steradians)? I notice Unreal Engine uses `cd / m²`, but it also requires a reference size (in meters) used for projection onto the scene. It would be helpful to know if other formats (`.hdr` or `.exr`) have chosen any existing precedent here.

Reply:
The way I think about it is that the output uses the same units meaning that the area represented is only resolved in the final step, usually in the display.
Modern displays knows how many candelas they can output and the same units are used for instance in HDR content.

Does this answer your question?

On a side note, this is one reason why I think it is important to proceed with https://github.com/KhronosGroup/glTF/pull/2083